api/graphql/mutations.js

```diff
@@ -118,3 +118,8 @@ export function unlinkAccount (userId, provider) {
   })
   .then(() => ({success: true}))
 }
+
+export function updateCommunitySettings (id, changes) {
+  return Community.find(id)
+  .then(community => community.update(convertGraphqlData(changes)))
```

Comment:
you need to validate that the user has permission to do this (i.e. is a moderator)